modules/pool/lib/apr-data-sources/fantom/stader-staked-ftm-apr.service.ts

```diff
@@ -0,0 +1,40 @@
+import { getAddress } from '@ethersproject/address';
+import { prisma } from '../../../../../prisma/prisma-client';
+import { PrismaPoolWithExpandedNesting } from '../../../../../prisma/prisma-types';
+import { TokenService } from '../../../../token/token.service';
+import { PoolAprService } from '../../../pool-types';
+
+export class StaderStakedFtmAprService implements PoolAprService {
+    constructor(private readonly tokenService: TokenService) {}
+
+    SFTMX_ADDRESS = getAddress('0xd7028092c830b5c8fce061af2e593413ebbc1fc1');
+
+    public async updateAprForPools(pools: PrismaPoolWithExpandedNesting[]): Promise<void> {
+        const tokenPrices = await this.tokenService.getTokenPrices();
+        const sftmxPrice = this.tokenService.getPriceForToken(tokenPrices, this.SFTMX_ADDRESS);
+        let operations: any[] = [];
+
+        for (const pool of pools) {
+            const sftmxBalance = pool.tokens.find((token) => token.address === this.SFTMX_ADDRESS)?.dynamicData
+                ?.balance;
+            if (sftmxBalance && pool.dynamicData) {
+                const sftmxPercentage = (parseFloat(sftmxBalance) * sftmxPrice) / pool.dynamicData.totalLiquidity;
+                const sftmxApr = 0.125 * sftmxPercentage;
```

Comment:
assume you checked that math with @danielmkm, would be good to add a comment here on where the math is based on. like why `0.125 * sftmxPercentage`

Reply:
copied from v1. Fair calculation of 12.5% APR from 13.5% APY I assume.

Reply:
ah so thats a hard percentage never changing then?

Reply:
I think it can change, if they change the staking rewards (which is under vote currently I think). But it's quite a hard percentage only changeable by gov votes.

Reply:
ah ok, well then lets ship it